app/pages/events/activity_delete.php

```diff
@@ -0,0 +1,37 @@
+<?php
+restrict_access(Access::$ADD_EVENTS);
+
+$activity_id = get_route_param("activity_id");
+$event_id = get_route_param("event_id");
+$activity = em()->find(Activity::class, $activity_id);
+if (!$activity) {
+    force_404("the activity of id $activity_id doesn't exist");
+    return;
```

Comment:
`return` here is not useful, since `force_404` calls `exit`

Reply:
👀